src/ui/components/UserReview/index.js

```diff
@@ -72,15 +77,17 @@ export const UserReviewBase = (props: InternalProps) => {
     showRating = false,
   } = props;
 
-  let body = reviewBody({ content: <LoadingText /> });
+  const showMoreId = review && review.id ? `${review.id}` : '0';
```

Comment:
I believe the fallback value should not be `'0'`, maybe `loading-text` would be better?